Makefile

```diff
@@ -139,7 +139,8 @@ test: pre-test
 
 .PHONY: run-integration-tests
 run-integration-tests: export VALUES_FILE=charts/fybrik/integration-tests.values.yaml
-run-integration-tests: export HELM_SETTINGS=--set "manager.solver.enabled=true"
+run-integration-tests: export HELM_SETTINGS=`--set "manager.solver.enabled=true" --set "coordinator.catalog=katalog" --set "opaServer.allowByDefault=false"`
```

Comment:
integration tests work with mockup services, use one of the notebook tests instead.